imapclient/setacl.go

```diff
@@ -0,0 +1,21 @@
+package imapclient
+
+import "github.com/emersion/go-imap/v2"
+
+// SetACL sends a SETACL command.
+func (c *Client) SetACL(mailbox string, identifier imap.RightsIdentifier, rights imap.RightSet) *SetACLCommand {
+	cmd := &SetACLCommand{}
+	enc := c.beginCommand("SETACL", cmd)
+	enc.SP().Mailbox(mailbox).SP().String(string(identifier)).SP().String(string(rights))
```

Comment:
Hm, it's a bit weird to pass an invalid `RightSet` here with a "+" or "-" prefix. Maybe we can have something like this instead?

```go
type RightModification byte

const (
	RightModificationReplace = RightModification(0)
	RightModificationAdd = RightModification('+')
	RightModificationRemove = RightModification('-')
)

func (*Client) SetACL(mailbox string, ri RightsIdentifier, rm RightModification, rs RightSet) *SetACLCommand
```

Does this make sense, do you have a different opinion?